spec/services/message_sender_spec.rb

```diff
@@ -1,83 +1,95 @@
 require "rails_helper"
 
 describe MessageSender do
-  it "creates a sent scheduled message if sent successfully" do
-    Timecop.freeze(Time.parse("10:00:00 UTC"))
-
-    scheduled_message = create(:scheduled_message)
-    employee = create(:employee)
-    client_double = Slack::Web::Client.new
-    slack_channel_id = "fake_slack_channel_id"
-    slack_channel_finder_double = double(run: slack_channel_id)
-
-    allow(Slack::Web::Client).to receive(:new).and_return(client_double)
-    allow(SlackChannelIdFinder).
-      to receive(:new).with(employee.slack_username, client_double).
-      and_return(slack_channel_finder_double)
-    allow(SentScheduledMessage).to receive(:create)
-
-    MessageSender.new(employee, scheduled_message).run
-
-    expect(SentScheduledMessage).to have_received(:create).with(
-      employee: employee,
-      scheduled_message: scheduled_message,
-      sent_on: Date.today,
-      sent_at: Time.parse("10:00:00 UTC"),
-      error_message: "",
-      message_body: scheduled_message.body,
-    )
-
-    Timecop.return
+  let(:base_dt) { Time.parse("10:00:00 UTC") }
```

Comment:
hey! do you mind including the setup for each test within the test itself -- similar to previous style? I generally try to stay away from `let` and `before` statements for these types of reasons: https://robots.thoughtbot.com/lets-not